pkg/server/server.go

```diff
@@ -200,6 +202,17 @@ func Run(logger log.Logger) error {
 		func(error) { serverInsecure.Shutdown(context.Background()) },
 	)
 
+	// We clean out old instances that have invalid tokens periodically as access
+	// to the PostgreSQL instances only relies on certificate authentication. This
+	// means that is situations, such as a user being offboarded, they will lose
+	// access to the draupnir, but not their instances.
+	instanceCleaner := instancecleaner.NewInstanceCleaner(logger, instanceStore, executor)
+
+	g.Add(
+		func() error { return instanceCleaner.Start(context.Background(), 30*time.Minute) },
```

Comment:
Going to add this as a flag so it can be tuned.

Reply:
How do you feel about:
```go
cleanerCtx, cleanerCancel := context.WithCancel(context.Background())
g.Add(..., func(error) { cleanerCancel() })
```

Reply:
Then you could remove the `done` channel from the cleaner

Reply:
Much simpler!